src/tests.rs

```diff
@@ -43,11 +43,13 @@ mod tests {
         version: &str,
         input_cargo_toml_path: &Path,
     ) -> Result<Option<String>, Box<dyn Error>> {
-        let mut crates_versions = get_version_mapping_with_fallback(crate::DEFAULT_GIT_SERVER, version)
-            .await
-            .unwrap();
+        let mut crates_versions =
+            get_version_mapping_with_fallback(crate::DEFAULT_GIT_SERVER, version)
```

Comment:
Please no no-op formatting changes, it makes reviewing slower.

Reply:
I am sorry about that, I will make sure I don't do it next time onwards.